packages/features/shell/Shell.tsx

```diff
@@ -787,7 +791,14 @@ function SideBar({ bannersHeight }: SideBarProps) {
         <div className="flex h-full flex-col justify-between py-3 lg:pt-6 ">
           <header className="items-center justify-between md:hidden lg:flex">
             <Link href="/event-types" className="px-2">
-              <Logo small />
+              {orgBranding ? (
+                <div className="flex items-center gap-2 font-medium">
+                  <Avatar alt="" imageSrc={orgBranding.logo} gravatarFallbackMd5="newTeam" size="sm" />
```

Comment:
Designs define this avatar to be 20x20 but `sm` size is 24x24, it looks good anyway.

Reply:
We may need to add some text to the uploader that we expect a 1:1 image.